tests/test_parse.py

```diff
@@ -6,12 +6,31 @@
 import pathlib
 import re
 from collections.abc import Callable
+from unittest.mock import patch
 
 # Third Party
 import pytest
 from assertpy import assert_that
 
-from instructlab.schema.taxonomy import TaxonomyMessageFormat, TaxonomyParser
+import instructlab.schema.taxonomy  # noqa: F401
+from instructlab.schema.taxonomy import TaxonomyMessageFormat, TaxonomyParser, TaxonomyReadingException
+
+
+class TestParserInit:
+    def test_parser_init_successful_when_schema_version_provided(self) -> None:
+        taxonomy = TaxonomyParser(schema_version=2)
+        assert_that(isinstance(taxonomy, TaxonomyParser)).is_equal_to(True)
+
+    def test_parser_init_successful_when_schema_version_not_provided_and_defaults_to_latest(self) -> None:
+        taxonomy = TaxonomyParser(schema_version=None)
+        assert_that(taxonomy.schema_version).is_equal_to(3)
```

Comment:
This will be brittle since it looks for a hard coded value of 3.
```suggestion
        assert_that(taxonomy.schema_version).is_equal_to(int(instructlab.schema.schema_versions()[-1].name[1:]))
```

Reply:
I can remove the test, but I don't think calling `instructlab.schema.schema_versions()` in this test will be effective. The purpose of this test is to confirm that `instructlab.schema.schema_versions()` returns the expected value.

Reply:
Yes, and the expected value is `int(instructlab.schema.schema_versions()[-1].name[1:])`.

Testing for a hard coded value will require this test to updated each time a schema version is added.

Reply:
Well, I think the purpose of the test is to confirm that when you pass schema_version=None to the TaxonomyParse constructor, that the schema_version is a the last version in schema_versions().

Reply:
Yeah, I agree the hardcoded value would make the test likely to break after v4 comes out.

I will update the test's assertion to your suggestion, but if we don't need the test, I can remove it.